pkg/prometheus/store.go

```diff
@@ -110,3 +114,25 @@ func AddScrapeClassesToStore(ctx context.Context, store *assets.StoreBuilder, na
 	}
 	return nil
 }
+
+func AddAlertmanagerEndpointsToStore(ctx context.Context, store *assets.StoreBuilder, namespace string, ams []monv1.AlertmanagerEndpoints) error {
```

Comment:
no need to move it to pkg/prometheus because it's only used by the server controller.

Reply:
Got it. Thanks.

Reply:
@simonpasquier  PTAL, thanks.